### DISCUSSION TITLE
RESTART & smoke & overwriting results

### DISCUSSION THREAD

**Discussion by Er9y714** [firemodels/fds#12350] [General]
Hi,

Is there a way to completely remove all the result between the restarted time and the end? I found a workaround with changing the CHID and restarting with RESTART_CHID, however it would be ideal to overwrite. 

Thanks.

The issue is related to these:
1. Once restarted and while running, I can still see all the old results in smokeview (confusing).
2. If there was a fire in the old calculation which produces smoke it is saved in .s3d files (expected). Then I edit fds file before a restart. I extinguish the fire in the edited fds file. When I restart and run I still see the smoke in the new calculated file smokeview.

**Comment by drjfloyd** [comment]
The purpose of restart is to continue a prior calculation from the point in time where the restart file was written. If in the real world you had a fire burning, and then the fire was suddenly extinguished, the smoke wouldn't also immeadiately disappear. It would take some time to dissippate. 

What is it you are trying to do with restart?

In smokeview you can set the time window that you want to load data for using the file/data/coloring dialog box. Simply set the start time to the restart time so you don't load data from before.

**Comment by Er9y714** [comment]
Thank you for the response. I should explain with an example. I will prepare a case soon. Thanks.

**Comment by Er9y714** [comment]
[test.zip](https://github.com/firemodels/fds/files/13873213/test.zip)
Please find the attached runs. 

Run1: regular run
Run2: ramp for burner VENT changed and VENT is deactivated after 10 s. So no soot production. This run is restarted so it rerun after 10s until the end. 

Please check the Run2 smv output of smoke. You can see the smoke is not in a behaviour of dispersing, it is identical to the first run.

**Comment by drjfloyd** [comment]
Converted this to an issue https://github.com/firemodels/fds/issues/12351.

**Comment by gforney** [comment]
let me know if you think this is a smokeview issue
